File: backend/src/main/java/com/project/dto/mood/response/MoodResponseDTO.java
package com.project.dto.mood.response;

import java.time.LocalDate;

import com.project.enums.Mood;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class MoodResponseDTO {
    private Long moodId;
    private Mood mood;
    private String notes;
    private LocalDate entryDate;

}

File: backend_dotnet/SafeMind.Core/DTOs/Mood/Response/MoodResponseDTO.cs
namespace SafeMind.Core.DTOs.Mood.Response
{
    public class MoodResponseDTO
    {
        public long MoodId { get; set; }
        public long UserId { get; set; }
        public SafeMind.Core.Enums.Mood Mood { get; set; }
        public string? Notes { get; set; }
        public DateTime Date { get; set; } // Representing the date of the mood
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
    }
}
